openrtx/include/core/ringbuf.hpp

```diff
@@ -26,17 +26,17 @@
 #endif
 
 #include <pthread.h>
+
 #include <cstdint>
 
 /**
  * Class implementing a statically allocated circular buffer with blocking and
  * non-blocking push and pop functions.
  */
-template < typename T, size_t N >
+template <typename T, size_t N>
 class RingBuffer
 {
-public:
```

Comment:
Is it possible to leave the `private`, `public` and `protected` fields with indentation zero (as they where before)?